docs/dev/pkgdown.yml

```diff
@@ -0,0 +1,15 @@
+pandoc: 2.9.2.1

```

Comment:
You don't need to make this change now, but I would recommend using a more recent version of `pandoc`.

You can install it from here:
https://pandoc.org/installing.html